docs/sources/mimir/operators-guide/architecture/deployment-modes/index.md

```diff
@@ -43,3 +43,34 @@ If you are interested in deploying Grafana Mimir in microservices mode, we recom
 [//]: # "Diagram source at https://docs.google.com/presentation/d/1LemaTVqa4Lf_tpql060vVoDGXrthp-Pie_SQL7qwHjc/edit#slide=id.g11658e7e4c6_1_53"
 
 ![Mimir's microservices mode](microservices-mode.svg)
+
+## Read-Write mode
+
+> **Warning:**
+> Read-Write deployment mode is currently considered experimental.
+
+The read-write mode provides a middle ground as an alternative to monolithic and microservices.
+
+In read-write mode, components are grouped into just three services to ease the operational overhead whilst still allowing scale to be tuned separately on the read and write paths. The services group the components as follows:
+
+1. read
+   - query-frontend
+   - querier
+2. backend
+   - store-gateway
+   - compactor
+   - ruler
+   - alertmanager
+   - query-scheduler
+   - overrides-exporter
+3. write
+   - distributor
+   - ingester
```

Comment:
```suggestion
- read
   - query-frontend
   - querier
- backend
   - store-gateway
   - compactor
   - ruler
   - alertmanager
   - query-scheduler
   - overrides-exporter
- write
   - distributor
   - ingester
```

Reply:
These are lists of items that do not need to be performed in a task. (Assume the reader can count in such a situation that is tempting to add numbers.)